specify -system flag to fix path to binaries used by tests
author: Kenneth Hoste (Ghent University)
--- Makefile.orig	2015-09-17 14:21:50.947592000 +0200
+++ Makefile	2015-09-17 14:22:33.015119930 +0200
@@ -134,7 +134,7 @@
 	@echo "| Test failed: ExternalAPI MPI not enabled"
 	@echo "--------------------------------------------------"; echo
    else
-	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/MPI/tests_enabling/data.pel resu_MPI -no_check_data
+	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/MPI/tests_enabling/data.pel resu_MPI -system Linux-$(CCC) -no_check_data
 	@ $(RM) resu_MPI
    endif
 
@@ -147,7 +147,7 @@
 	@echo "| Test failed: ExternalAPI MPI not enabled"
 	@echo "--------------------------------------------------"; echo
    else
-	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/MPI/tests_enabling/data.pel resu_MPI -no_check_data -opt2
+	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/MPI/tests_enabling/data.pel resu_MPI -system Linux-$(CCC) -no_check_data -opt2
 	@ $(RM) resu_MPI
    endif
 	@ cd $(APPLI_DIR) ; $(MAKE) test_install CCC=$(CCC) OPT_TEST=$(OPT_TEST)
@@ -161,7 +161,7 @@
 	@echo "| Test failed: ExternalAPI METIS not enabled"
 	@echo "--------------------------------------------------"; echo
    else
-	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/METIS_4.0.1/tests_enabling/data.pel resu_METIS -no_check_data
+	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/METIS_4.0.1/tests_enabling/data.pel resu_METIS -system Linux-$(CCC) -no_check_data
 	@ $(RM) resu_METIS
    endif
 
@@ -174,7 +174,7 @@
 	@echo "| Test failed: ExternalAPI METIS not enabled"
 	@echo "--------------------------------------------------"; echo
    else
-	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/METIS_4.0.1/tests_enabling/data.pel resu_METIS -no_check_data -opt2
+	@ $(BIN_DIR)/isis $(PELICANS_DIR)/etc/ExternalAPI/METIS_4.0.1/tests_enabling/data.pel resu_METIS -system Linux-$(CCC) -no_check_data -opt2
 	@ $(RM) resu_METIS
    endif
 
